js/app/src/components/Dataset/Dataset.svelte

```diff
@@ -156,8 +156,14 @@
 							on:mouseleave={() => handle_mouseleave()}
 						>
 							{#each sample_row as { value, component }, j}
-								{#if components[j] !== undefined && component_map[components[j]] !== undefined}
-									<td>
+								{@const component_name = components[j]}
+								{#if component_name !== undefined && component_map[component_name] !== undefined}
+									<td
+										style="max-width: {component_name === 'textbox'
+											? '35ch'
+											: 'auto'}"
+										class={component_name}
+									>
```

Comment:
We need to set a max-width for the textbox _container_ (this table cell) to ensure it doesn't end up as one long line and remains readable. 35ch is 35 characters which keeps things nice and readable without taking up too much space.